test/spec/CMakeLists.txt

```diff
@@ -12,45 +12,15 @@ FetchContent_Declare(
 FetchContent_MakeAvailable(wasmedge_unit_test)
 message(STATUS "Downloading the WASM spec test suite -- done")
 
-find_package(RapidJSON QUIET)
-if(NOT RapidJSON_FOUND)
-  message(STATUS "Downloading RapidJSON source")
-  FetchContent_Declare(
-    RapidJSON
-    GIT_REPOSITORY https://github.com/Tencent/rapidjson.git
-    GIT_TAG        master
-    GIT_SHALLOW    TRUE
-    )
-
-  get_property(
-    compile_options
-    DIRECTORY
-    PROPERTY COMPILE_OPTIONS
-    )
-  set_property(
-    DIRECTORY
-    APPEND
-    PROPERTY COMPILE_OPTIONS
-    -Wno-conditional-uninitialized
-    -Wno-implicit-int-conversion
-    -Wno-zero-as-null-pointer-constant
-    )
-  unset(compile_options)
-  set(RAPIDJSON_BUILD_DOC OFF CACHE BOOL "Build RapidJSON documentation." FORCE)
-  set(RAPIDJSON_BUILD_EXAMPLES OFF CACHE BOOL "Build RapidJSON examples." FORCE)
-  set(RAPIDJSON_BUILD_TESTS OFF CACHE BOOL "Build RapidJSON perftests and unittests." FORCE)
-  set(RAPIDJSON_BUILD_CXX11 OFF CACHE BOOL "Build RapidJSON with C++11" FORCE)
-  set(RAPIDJSON_BUILD_CXX17 ON CACHE BOOL "Build RapidJSON with C++17" FORCE)
-  FetchContent_MakeAvailable(RapidJSON)
-  message(STATUS "Downloading RapidJSON source -- done")
-endif()
-
-add_library(RapidJSON INTERFACE)
-if(RAPIDJSON_INCLUDE_DIRS)
-  target_include_directories(RapidJSON INTERFACE ${RAPIDJSON_INCLUDE_DIRS})
-else()
-  target_include_directories(RapidJSON INTERFACE ${RapidJSON_SOURCE_DIR}/include)
-endif()
+message(STATUS "Downloading SIMDJSON")
+include(FetchContent)
```

Comment:
Since some distro will provide simdjson package, we should keep the `find_package()` to get simdjson from the system.

Reply:
Ok, will do

Reply:
I have made the changes in [f07568f](https://github.com/WasmEdge/WasmEdge/pull/2659/commits/f07568ff4216c571014bd3dae3e3d4f41982074a), and tested it on my system successfully.